src/cppsim/circuit.cpp

```diff
@@ -304,8 +304,8 @@ std::string QuantumCircuit::to_string() const {
     UINT max_block_size = 0;
 
     for (const auto gate : this->_gate_list) {
-        UINT whole_qubit_index_count = (UINT)(
-            gate->target_qubit_list.size() + gate->control_qubit_list.size());
+        UINT whole_qubit_index_count = (UINT)(gate->target_qubit_list.size() +
+                                              gate->control_qubit_list.size());
```

Comment:
Code format is slightly changed due to the transition from clang-format 10 to clang-format 14.
clang-format 20 is not standard on Ubuntu 24.04.